### DISCUSSION TITLE
Permission for iModel Export

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/itwinjs-core#2913] [Q&A]
Hello - I am trying to develop and debug the FutureOn-Bentley Connector (imodel-bridge) application. 
So I would like to be able to export a text readable version of the iModel that is generated by the synchronisation.
I have tried to use the itwin export API (https://developer.bentley.com/apis/export/operations/start-export/), however when I invoke the API I get an error response:
403 Forbidden
{
    "error": {
        "code": "ResourceAccessForbidden",
        "message": "User is not authorized to access resource."
    }
}

My question is - how can I change my permissions to be able to export an iModel?

**Comment by rseydler** [comment]
Hi. 

Just checking but before you can use the export you need to store the users token.

Authorization
The export process usually takes time and is performed in the background. For that, we need to store the user's refresh token. You can get it by using Get Authorization Information API. This API will return the current status and a redirect URL if the token has to be renewed.

Info taken from https://developer.bentley.com/apis/export/overview/
